e2e/commands/import.e2e.js

```diff
@@ -1571,6 +1571,107 @@ describe('bit import', function () {
     });
   });
 
+  describe('component with shared directory across files and dependencies', () => {
```

Comment:
maybe we should add a case when the user add another file to existing component which makes the shared lib to change?